### PR TITLE
tools: add ethtool info to fpagainfo

### PR DESCRIPTION
**Pull request by anandaravuri**
- get list of eth interfaces in system
 - check interfaces belongs to n3000 fpga
 - run eth tool command to print phy group status

### CODE REVIEW

File: tools/libboard/board_n3000/board_n3000.c [hunk 1]
```diff
@@ -295,17 +308,90 @@ fpga_result print_board_info(fpga_token token)
...
+				// Interface up
+				memset(cmd, 0, sizeof(cmd));
+				if (snprintf(cmd, sizeof(cmd),
+					"%s %s %s", IFCONFIG_STR, intf->if_name,
+					IFCONFIG_UP_STR) < 0) {
+					OPAE_ERR("snprintf failed");
+					res = FPGA_EXCEPTION;
+					goto out_free;
+				}
+				printf("cmd:%s \n", cmd);
```
Comment (tswhison) [inline tools/libboard/board_n3000/board_n3000.c:349]: Remove

File: tools/libboard/board_n3000/board_n3000.c [hunk 2]
```diff
@@ -295,17 +308,90 @@ fpga_result print_board_info(fpga_token token)
...
+				}
+				// eth tool command
+				memset(cmd, 0, sizeof(cmd));
+				if (snprintf(cmd, sizeof(cmd),
+					"%s %s", ETHTOOL_STR, intf->if_name) < 0) {
+					OPAE_ERR("snprintf failed");
+					res = FPGA_EXCEPTION;
+					goto out_free;
+				}
+				printf("cmd:%s \n", cmd);
```
Comment (tswhison) [inline tools/libboard/board_n3000/board_n3000.c:363]: Remove

File: tools/libboard/board_n3000/board_n3000.c (hunk 1 above)
Comment (anandaravuri) [reply tools/libboard/board_n3000/board_n3000.c]: fixed, removed printf

File: tools/libboard/board_n3000/board_n3000.c (hunk 2 above)
Comment (anandaravuri) [reply tools/libboard/board_n3000/board_n3000.c]: fixed, removed printf

**Pull request merged**